src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -85,7 +94,6 @@ public UpdatePullRequestCommentsSteps(GitHubService gitHubService, GitHubPropert
         this.flowProperties = flowProperties;
         this.cxProperties = cxProperties;
         this.scaProperties = scaProperties;
-
     }
 
     private void initSca() {
```

Comment:
This method is already duplicated in 4 step classes. I suggest that we extract it into a common method.